vscode/microsoft-kiota/src/steps.ts

```diff
@@ -162,14 +162,19 @@ export async function generateSteps(existingConfiguration: Partial<GenerateState
             case 'plugin':
                 return inputPluginName;
             case 'apimanifest':
```

Comment:
```suggestion
```

If we go with the switch case provided in `transformToGenerationConfig` where generationType for apimanifest and openai are first mapped to 'Other', we wouldn't need an additional case for apimanifest.